src/Core/Grid/Definition/Factory/AbstractGridDefinitionFactory.php

```diff
@@ -62,6 +62,10 @@
             'definition' => $definition,
         ]);
 
+        \Hook::exec('action' . $definition->getId() . 'GridDefinitionModifier', [
```

Comment:
Why do we use both `$this->hookDispatcher->dispatchWithParameter()` and `\Hook::exec()` ? Should not it be either one or the other ?

Reply:
using `\Hook::exec()` works as expected. :D but i unintentionally left it here when playing with hooks, gonna remove it. :)